tests/playwright/specs/compare/compare-article-download.spec.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+'use strict';
+
+const { test, expect } = require('@playwright/test');
+const openPage = require('../../scripts/open-page');
+const url = '/en-US/compare/chrome';
```

Comment:
Added for an automated check on the conditional download link display on compare article pages. Should I be looping through all the compare article pages or is single check enough?

Reply:
Given it's in the base template, I think there's not much value in a per-article check